.github/workflows/deployment.yml

```diff
@@ -279,10 +279,10 @@ jobs:
           password: ${{ secrets.ACR_PASSWORD }}
 
       - name: Artifacts 🗃️
-        working-directory: ./azure-functions/${{ matrix.functions }}
+        working-directory: .
```

Comment:
does this mean we don't even need this is we're just setting it to '.'? Unsure if it's resetting something, either way LGTM I'll approve!

Reply:
Do you mean we don't need `working-directory` argument?

Reply:
yeah